# HG changeset patch
# User John Donoghue
# Date 1428500279 14400
#      Wed Apr 08 09:37:59 2015 -0400
# Node ID ae12edf69bce48e4ccacf51dc7d85b007de3d006
# Parent  49df5054c02ede4a1f80da84e0a006153852e4e8
Set default uimenu position property to 0 (Bug #44770)

* libinterp/corefcn/graphics.in.h: set postion property from 9 to 0

diff --git a/libinterp/corefcn/graphics.in.h b/libinterp/corefcn/graphics.in.h
--- a/libinterp/corefcn/graphics.in.h
+++ b/libinterp/corefcn/graphics.in.h
@@ -5311,17 +5311,17 @@ public:
     BEGIN_PROPERTIES (uimenu)
       any_property __object__ , Matrix ()
       string_property accelerator , ""
       callback_property callback , Matrix ()
       bool_property checked , "off"
       bool_property enable , "on"
       color_property foregroundcolor , color_values (0, 0, 0)
       string_property label , ""
-      double_property position , 9
+      double_property position , 0
       bool_property separator , "off"
       // Octave-specific properties
       string_property fltk_label h , ""
     END_PROPERTIES
 
   protected:
     void init (void)
     { }
